lib/forms/CoBrALab-Mouse-MRI-Form/index.ts

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable perfectionist/sort-objects */
+
+import { defineInstrument } from '/runtime/v1/@opendatacapture/runtime-core'
+import { z } from '/runtime/v1/zod@3.23.x'
+
+
+const scanNameOptions = {
```

Comment:
you ask PV5/PV6, do you have all the old PV5 sequences that might be used?

Reply:
from the list dan sent the only pv5 protocol still used is the Ex-vivo protocol, which includes scan names 01-Tripilot-multi and exvivoDanFLASH